pages/css/formatting.md

```diff
@@ -3,84 +3,10 @@ title: Formatting
 permalink: /css/formatting/
 sidenav: css
 ---
-# Formatting
-## Spacing
-- Where possible, limit CSS files’ width to 80 characters. See
-  [notes](#format_notes) to see how to configure your text editor to 80
-  characters.
-  - There will be unavoidable exceptions to this rule, such as URLs, or
-    gradient syntax. Don’t worry.
-- Use soft-tabs with a two space indent.
-- Put one space after `:` in property declarations.
-- Put spaces before `{` in rule declarations.
-- Put a blank line between each selector block.
-- To close a selector block, put an unindented closing curly brace on a
-  separate line.
-- Each declaration should appear on its own line for more accurate error
-  reporting.
-- Do not indent selectors.
-
-```scss
-// Bad
-.rule{
-    margin:3px;text-align:center;}
-
-// Good
-.rule {
-  margin: 3px;
-  text-align: center;
-}
-
-.another_rule {
-  margin: 3px;
-}
-```
-
-Multiple selectors should each be on a single line, with no space after each
-comma, unless the selector is less than five chars.
-
-```scss
-// Bad
-selector1, selector2 {
-}
-
-// Good
-selector1,
-selector2,
-selector3 {
-}
-
-// Good
-h1, h2 {
-}
-```
-
-## Property-value pairs
-- Put each pair on its own line.
-- Indent each pair one level.
-- End in a semicolon.
-
-```scss
-selector {
-  name: value;
-  name: value;
-}
-```
-
-Spaces should separate values and operators in Sass expressions.
 
-```scss
-// Bad
-selector {
-  font-size: ($font-size+2em);
-  font-size: ($font-size +2em);
-}
+# Formatting
 
-// Good
-selector {
-  font-size: ($font-size + 2em);
-}
-```
+We recommend using [Prettier](https://prettier.io), and enabling it in your editor by default.
```

Comment:
Per the efforts of the documentation working group, we're trying to document why we make a recommendation and note how strong that recommendation is. Could you take a stab at that here?